packages/pigeon/platform_tests/android_unit_tests/android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -5,7 +5,7 @@ connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
 connection.project.dir=
 eclipse.preferences.version=1
 gradle.user.home=
-java.home=/Library/Java/JavaVirtualMachines/jdk-11-latest/Contents/Home
+java.home=/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home
```

Comment:
Should this file actually be checked it? It seems to have paths on your machine.

Reply:
It isn't ignored by our default `.gitignore` settings.  This isn't the only package that has this file included.  I'm not sure where it gets used.  I know I don't use eclipse.  I worry that the tool is updating it but also assuming that it exists.